app/views/sign_in_security_check_failed/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<% self.title = t('security_check_failed.title') %>
+
+<%= render StatusPageComponent.new(status: :warning) do |c| %>
+  <% c.with_header { t('security_check_failed.title') } %>
+  <p>
+    <%= t('security_check_failed.details') %>
+    <%= new_tab_link_to(
+          t('security_check_failed.learn_more', app_name: APP_NAME),
+          help_center_redirect_path(
+            category: 'trouble-signing-in',
+            article: 'security-check-failed',
```

Comment:
These articles need to be added to an allowlist to work:

https://github.com/18F/identity-idp/blob/74d2492c819b30e42c15b85a350411753784fd64/app/services/marketing_site.rb#L8-L20